spec/takeaway_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'takeaway'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu: menu) }
+
+  let(:menu) { double(:menu, print: printed_menu) }
```

Comment:
saw this else where, does this make a double of the menu class and method print ?